### PR TITLE
Bug: Minor refactor transaction

### PR DESCRIPTION
**Pull request by magestrio**
## Type

- [ ] Feature
- [X] Bug fix
- [ ] CI/CD
- [ ] Documentation
- [ ] Other

## Description

Brief description of the changes made.

### CODE REVIEW

File: packages/zkdb/src/types/transaction.ts [hunk 1]
```diff
@@ -11,6 +11,6 @@ export type TDbTransaction = {
   databaseName: string;
```
Comment (SangTran-127) [inline packages/zkdb/src/types/transaction.ts:11]: export type TTransaction

Comment (chiro-hiro) [reply packages/zkdb/src/types/transaction.ts:11]: Should move all type definition to `@zkdb/common` don't define type here.

Comment (magestrio) [reply packages/zkdb/src/types/transaction.ts:11]: done

File: packages/serverless/src/apollo/app/transaction.ts [hunk 2]
```diff
@@ -87,11 +91,16 @@ const transactionConfirm = authorizeWrapper<
...
   Joi.object({
     databaseName,
-    id: Joi.string().required(),
+    transactionObjectId: Joi.string().required(),
     txHash: Joi.string().required(),
   }),
   async (_root: unknown, args: TTransactionConfirmRequest, ctx) =>
-    await Transaction.confirm(args.databaseName, ctx.userName, args.txHash)
+    await Transaction.confirm(
+      args.databaseName,
```
Comment (SangTran-127) [inline packages/serverless/src/apollo/app/transaction.ts:99]: no need await

**Review by chiro-hiro — APPROVED** [review]

**Pull request merged**